src/dashboard/src/components/administration/migrations/0007_auto_20161116_1728.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('administration', '0006_use_statement_optional'),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
This migration should be squashed with the commit that introduced the change before merging.